.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: publish
+
+on:
+  release:
+    types: [created] # Trigger when release is created
```

Comment:
According to https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release this should only fire when release actions happen. However I changed the action from `created` to `published` to avoid pre-releases being published.